README.md

```diff
@@ -1,2 +1,31 @@
 # sentry-fullstory
 The Sentry-FullStory integration creates a link from the Sentry Error to the FullStory replay. It also creates a link from the FullStory event to the Sentry error.
+
+## Pre-Requisites
+
+For the Sentry-FullStory integration to work, you must have the [Sentry browser SDK package](https://www.npmjs.com/package/@sentry/browser) and FullStory running. 
+
+## Installation
+To install the stable version:
+```
+npm install --save sentry-fullstory
+```
+This assumes you are using npm as your package manager.
```

Comment:
I've noticed that it's also common to put the yarn instructions. So, how about this:

To install and save in your package.json dependencies, run:
```
// with npm
npm install --save sentry-fullstory

// with yarn
yarn add sentry-fullstory
```